src/open_inwoner/templates/pages/plans/note_edit.html

```diff
@@ -0,0 +1,28 @@
+{% extends 'master.html' %}
+{% load i18n button_tags card_tags map_tags form_tags grid_tags %}
+
+{% block extra_head %}
+  {{ form.media }} {# Loads necessary JS for CKEditor #}
+{% endblock %}
+
+{% block content %}
+  <h1 class="utrecht-heading-1">{% trans "Notitie maken en aanvullen" %}</h1>
+  <p class="utrecht-paragraph">
+    {% trans "Voeg een notitie toe en/of vul een bestaande notitie aan met nieuwe tekst." %}
+  </p>
+
+  {% render_grid %}
+    {% render_column start=4 span=7 %}
+      <div class="plan__note">
+        {% render_form id="note-edit" form=form method="POST" %}
+          {% csrf_token %}
+          {% input form.note %} {# TODO: Remove the javascript that hides the CKeditor notification when switching to different text-editor #}
```

Comment:
If you use the JS solution and switch over to CSS.
```suggestion
          {% input form.note %}
```

Reply:
No.